### ISSUE TITLE
talos install

### ISSUE THREAD

**Issue by wsy0101** [siderolabs/sidero#1206]
Hello, I encountered a confusion while deploying a cluster for Sidero metal. I hope you can provide me with an answer. The description of the problem is as follows:

I have deployed a set of sidero metal in the K8S environment on the public network. Can this sidero metal perform talos installation on bare metal in the private network (a local area network that can access the public network)?

**Comment by smira** [comment]
yes, it can, Sidero Metal is designed for a private data center scenarios.

It might be better to use discussions for questions.
